cmd/webhook/main.go

```diff
@@ -286,6 +294,10 @@ func NewConversionController(ctx context.Context, cmw configmap.Watcher) *contro
 }
 
 func main() {
+	sbSelector := psbinding.WithSelector(psbinding.ExclusionSelector)
+	if os.Getenv("SINK_BINDING_OPT_OUT_SELECTOR") == "true" {
```

Comment:
This is a confusing name.

Reply:
I would change this to `SINK_BINDING_REQUIRE_INCLUSION_SELECTOR`

Reply:
I have also problems with this name, tbh

so the `SINK_BINDING_OPT_OUT_SELECTOR` is defaultuing to `false` in the yaml.

So, I think think, because OPT_OUT is false, that it is not "opt out", so included.  

but with setting the value to true (I want to OPT_OUT), the `InclusionSelector` is used ? :confused:

Reply:
So, basically the configuration for the mutation webhook is "I WANT TO OPT OUT"  -> false  

but, if a user wants to opt_out, she has to set it to `true`...

/cc @liu-cong can we capture this _very_ explicit in the docs ? 

Please ping me on the PR for the docs repo

Reply:
What about just `SINK_BINDING_SELECTOR` and let the user pick `inclusion` or `exclusion`? See PR #2662.